### PR TITLE
Update remaining libraries

### PR DESCRIPTION
**Pull request by s-l-teichmann**
The remaining libraries are now able to be updated.

Solves #345

### CODE REVIEW

**Review by JanHoefelmeyer — APPROVED** [review]
Works for me

**Review by tschmidtb51 — COMMENTED** [review]
Except for the comment - LGTM.

@s-l-teichmann I guess you merge only this one instead of #349? As both seem to have a set of shared commits...

File: .github/workflows/itest.yml [hunk 1]
```diff
@@ -11,11 +11,17 @@ jobs:
         with:
           go-version: 1.19.1
 
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
```
[the remark this answers is not in the document]
Comment (tschmidtb51) [reply .github/workflows/itest.yml:18]: Answered in https://github.com/csaf-poc/csaf_distribution/pull/349#discussion_r1132116290

**Comment by tschmidtb51** [comment]
@s-l-teichmann: I resolved the merge conflict with `main` that came from updating 
```
golang.org/x/crypto v0.6.0 h1:qfktjS5LUO+fFKeJXZ+ikTRijMmljikvG68fpMMruSc=
golang.org/x/crypto v0.6.0/go.mod h1:OFC/31mSvZgRz0V1QTNCzfAI1aIRzbiufJtkMIlEp58=  
```
to
```
golang.org/x/crypto v0.7.0 h1:AvwMYaRytfdeVt3u6mLaxYtErKYjxA2OXjJ1HHq6t3A=
golang.org/x/crypto v0.7.0/go.mod h1:pYwdfH91IfpZVANVyUOhSIPZaFoJGxTFbZhFTx+dXZU=
```

**Review by tschmidtb51 — APPROVED** [review]
LGTM. I run the integration test on Github and it passed. I didn't check locally.

**Pull request merged**